databricks/util/reflect_resource_test.go

```diff
@@ -0,0 +1,140 @@
+package util
```

Comment:
there's a test for that in other branch

Reply:
merged the test files to capture the tests